src/dst/todst.cpp

```diff
@@ -1151,11 +1151,14 @@ static Expr *dst_interpolate(CSTElement intp) {
   std::string total;
 
   MultiLineStringIndentationFSM fsm;
-  for (CSTElement i = intp.firstChildNode(); !i.empty(); i.nextSiblingNode()) {
-    if (args.size() % 2 == 0) fsm.accept(i);
+  CSTElement i = intp.firstChildNode();
+  while (!i.empty()) {
+    fsm.accept(i);
+    i.nextSiblingNode();
+    i.nextSiblingNode();
```

Comment:
Here we're unconditionally skipping the next two sibling nodes. What do we expect these nodes to be? And can a comment about this be added?